translator/sample files/navjoy_data/navjoy_form568.json

```diff
@@ -0,0 +1,2152 @@
+[
```

Comment:
I don't see this file referenced anywhere, are we using it?

Reply:
removed